app/Console/Commands/ForceDeletePosts.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+use Rogue\Managers\PostManager;
+
+class ForceDeletePosts extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:forceDeletePosts
+                            {type}
+                            {source}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Force deletes posts in Rogue by type and source';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(PostManager $postManager)
+    {
+        // Post Manager Instance.
+        $this->postManager = $postManager;
+
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        if ($this->confirm('Are you sure you want to delete posts of type ' . $this->argument('type') . 'from ' . $this->argument('source'))) {
```

Comment:
It'd be nice to write this as a guard clause so we follow the [golden path](https://en.wikipedia.org/wiki/Happy_path) (rather than indenting the rest of this function)!

```php
if (! $this->confirm('Are you sure you want to delete posts of type ' . $this->argument('type') . 'from ' . $this->argument('source'))) {
    return;
}

info('rogue:forceDeletePosts: Starting to force delete posts by type and source.');
$posts = Post::where(...);
```

Reply:
ohhh I love this @DFurnes ! I've never heard of the golden/happy path but I love this idea and is a lot easier to read. Thanks for sharing! Updated :)